code/datums/traits/negative.dm

```diff
@@ -579,3 +579,44 @@
 	lose_text = "<span class='notice'>Your mind finally feels calm.</span>"
 	medical_record_text = "Patient's mind is in a vulnerable state, and cannot recover from traumatic events."
 	hardcore_value = 9
+
+/datum/quirk/allergic
+	name = "Extreme Medicine Allergy"
+	desc = "Ever since you were a kid you always were allergic to certain chemicals..."
+	value = -2
+	gain_text = "<span class='danger'>You feel your immune system shift.</span>"
+	lose_text = "<span class='notice'>You fell your immune system phase back into perfect shape.</span>"
+	medical_record_text = "Patient's immune system responds violently to certain chemicals."
```

Comment:
this should say what chemical you're allergic to, because that's medical information that should be on a medical database

Reply:
i know..i looked it up. that was my original idea, but medical records dont uh use that variable... at all

Reply:
i guess i will put it there aswell, but the information is already given in the name

Reply:
if it was coded well you should be able to change the med_record text onspawn to show your allergens.

Reply:
well i did do that, problem is for some reason medical_record_text isnt used anywhere outside of quirks. I mean it.